### ISSUE TITLE
(encrypted) snapshots not found/shown on new machine

### ISSUE THREAD

**Issue by deesnook** [bit-team/backintime#556]
I'm using Back In Time 1.1.12 in elementary 0.3.2 (based on Ubuntu 14.04) to backup my home folder to an external hard drive. Which seems to have worked fine so far.

Now I've bought a new computer and installed the system from scratch. Of course I want to restore the data from the home folder backup using Back In Time. But alas, none of the ~15 snapshots on my external drive shows up in the snapshot list. I've tried to restore the configuration on first start of BIT, but it doesn't work because no configuration/snapshot from my external HDD can be detected. The backup is encrypted. The restore configuration menu says in that case I have to mount a snapshot manually to detect the configuration, but how can I do that if all the folder names of the backup are just some random gibberish because of the encryption?

I was told in a forum to copy over the config file referring me to this answer: http://askubuntu.com/questions/509857/transfer-backintime-saves-from-old-pc-to-new-one

What I don't get about this hint is: How would I copy over the config file if my HDD breaks down and I thus cannot access the config file anymore? And isn't just this the very situation I do a backup for in the first place?

Any help will be greatly appreciated! Thank you!

**Comment by Germar** [comment]
The config file is stored with every snapshot. So you can restore it from the snapshot. But you're right, if your snapshots are encrypted there is no straight way to do this.

You'd need to mount the encrypted snapshots manually with `mkdir ~/mnt && encfs /path/to/backup ~/mnt` to find the unencrypted version in `~/mnt/backintime/HOST/USER/PROFILE_ID/last_snapshot`.

I'll have a look if I can backup the config in plain text beside the encrypted folder, too.

**Comment by deesnook** [comment]
Thanks Germar for the quick reply!! 

I was able to work around the problem by manually defining the snapshot path on the new machine according to the path I had used on the old machine. But you're right, from my simplistic user's perspective I'd prefer if BIT just detected encrypted snapshots (and maybe asked me if I wanted to restore them with a password prompt).

If you included the config file in plain text beside the encrypted folder, that plain text wouldn't include the encryption password, would it? (stupid question I know because if it did the encryption would be useless)
